packages/core/src/animation/Animator.ts

```diff
@@ -443,7 +507,10 @@ export class Animator extends Component {
   }
 
   private _updateLayer(layerIndex: number, firstLayer: boolean, deltaTime: number, aniUpdate: boolean): void {
-    let { blendingMode, weight } = this._animatorController.layers[layerIndex];
+    let { blendingMode, weight, stateMachine } = this._animatorController.layers[layerIndex];
+
+    if (!stateMachine) return;
```

Comment:
Before have bug?

Reply:
animatorLayer add stateMachine in constructor